File: src/Generated20260425.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260425 {
public:
    // Generated class 20260425_090111_677583
    int value = 379;
    string created = "20260425_090111_677583";
    
    int process() {
        int result = value;
        for (int i = 0; i < 19; i++) {
            result += i * 9;
        }
        return result;
    }
};

int main() {
    Generated20260425 obj;
    cout << obj.process() << endl;
    return 0;
}

File: src/Generated20260425.go
package main

import "fmt"

type Generated20260425 struct {
    // Generated struct 20260425_090002_368050
    Value int
    Created string
}

func (c *Generated20260425) Process() int {
    result := c.Value
    for i := 0; i < 6; i++ {
        result += i * 2
    }
    return result
}

func main() {
    obj := Generated20260425{Value: 445, Created: "20260425_090002_368050"}
    fmt.Println(obj.Process())
}